packages/embed/src/components/pausedpopover/AudiusLogo.module.css

```diff
@@ -1,6 +1,6 @@
 .container {
   background-size: contain;
   background-repeat: no-repeat;
-  width: 196px;
+  width: 208px;
```

Comment:
🤔  I assume this changed from the design

Reply:
yup